keps/sig-node/3619-supplemental-groups-policy/README.md

```diff
@@ -907,6 +969,8 @@ and creating new ones, as well as about cluster-level services (e.g. DNS):
       - Impact of its degraded performance or high-error rates on the feature:
 -->
 
+Specific version of CRI.
```

Comment:
Versions of container runtimes supporting the feature.

Reply:
```suggestion
Container runtimes supporting [CRI api v0.31.0](https://github.com/kubernetes/cri-api/tree/v0.31.0) or above.

For example, 
- containerd: v2.0 or later
- CRI-O: v1.31 or later
```